precompiles/vesting/tx.go

```diff
@@ -124,7 +124,7 @@ func (p *Precompile) FundVestingAccount(
 		}
 
 		// NOTE: This ensures that the changes in the bank keeper are correctly mirrored to the EVM stateDB.
-		amt := vestingCoins.AmountOf(utils.BaseDenom).BigInt()
+		amt := vestingCoins.AmountOf(evmostypes.BaseDenom).BigInt()
```

Comment:
why don't we get the evm denom from the config here?

Reply:
this shouldn't use the base denom, it should check whether the coin is the same as the native EVM  denom and then update only if that's the case